app/src/main/java/com/android/unio/ui/event/EventCard.kt

```diff
@@ -73,7 +71,7 @@ fun EventCard(
     return
   }
 
-  var isSaved by remember { mutableStateOf(user!!.savedEvents.contains(event.uid)) }
```

Comment:
I see why you can remove the mutableState as it is always trigger by the onClick, good job !